plugin-name/plugin-name.php

```diff
@@ -66,10 +59,5 @@ function deactivate_plugin_name() {
  *
  * @since    1.0.0
  */
-function run_plugin_name() {
-
-	$plugin = new Plugin_Name();
-	$plugin->run();
-
-}
-run_plugin_name();
+$plugin = new Plugin_Name();
```

Comment:
I don't see the issue with the global functions to begin with. You're not supposed to leave them as `plugin_name`; you change them to the name of your plugin, which, assuming your plugin name doesn't conflict with another plugin out there (which it shouldn't anyway), there will be no conflict.